src/main/resources/story/walmart/CreateUser.story

```diff
@@ -0,0 +1,16 @@
+Description: Story to test user creation;
+
+Scenario: Open Walmart main page
+Given I am on a page with the URL 'https://www.walmart.com/'
+Then the page title is 'Walmart.com | Save Money. Live Better.'
+
+Scenario:  Create user
+When I click on an element by the xpath '//button[@aria-label='Your Account']'
+When I click on an element by the xpath  '//div[contains(text(),'Create Account')]' 
```

Comment:
You can replace it with a step that refers to the text specifically  'When I click on an element with the text ''